subsys/bluetooth/mesh/Kconfig.models

```
@@ -653,11 +653,14 @@ menuconfig BT_MESH_SCENE_SRV
 
 config BT_MESH_SCENES_MAX
 	int "Max number of scenes"
-	default 4
+	default 16
 	range 1 65535
 	depends on BT_MESH_SCENE_SRV
 	help
-	  Max number of scenes that can be stored by a single Scene Server.
+	  Maximum number of scenes that can be stored by a single Scene Server.
+	  Please note that the specification suggests that this value should be
+	  16. MshMdlv1.0.1: "The Scene Register state is a 16-element array of
+	  16-bit values representing a Scene Number."
```

"specification suggests" is too soft wording I think.

```suggestion
	  Maximum number of scenes that can be stored by a single Scene Server.
	  Note that, according to the specification, a Scene Server can store 16
	  elements. MshMdlv1.0.1: "The Scene Register state is a 16-element array
	  of 16-bit values representing a Scene Number."
```